Src/BootCamp.Chapter/Teachers/ITeacher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5789960)